ONSdigital/eq-author-terraform
# Configure author terraform to work with DynamoDB.

Some changes were required to author's terraform scripts to support running author against a DynamoDB data store.

### How to test
`terraform init --upgrade`
`terraform plan`
`terraform apply`
Ensure you can create a questionnaire in author and launch a preview in runner.
Log into the AWS dev console and confirm that dynamo db tables exist for your environment.
Tables should contain the created questionnaire version.
